challenges/frontend/challenge.md

```diff
@@ -0,0 +1,157 @@
+# Application Developer (front-end)
+
+## Summary
+Implement a simple application that allows a user to browse directory content on a remote server.
+
+## Rationale
+We would like to evaluate your skill in the following areas:
+- Taking existing high-level requirements and translating them to a functional application
+- Writing production level code that does not depend on gigabytes of npm packages
+- Communicating with the team when working on the challenge
+- Handling feedback from Teleport employees
```

Comment:
Handling feedback. I would also finish every sentence with a dot here and below.